pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "connect-telegram-bot"
-version = "0.1.0"
+version = "0.1.1"
```

Comment:
no need to change and if we do, let's pair with tag, 1.0.1 will be this one

Reply:
fixed